src/js/services/txFormatService.js

```diff
@@ -37,6 +37,7 @@ angular.module('copayApp.services').factory('txFormatService', function(profileS
       }, 0);
     }
 
+    tx.toAddress = tx.outputs[0].toAddress; // Get first address for single transactions
```

Comment:
You are right. It 's not always defined for old tx proposals. Fixing it right now.